cluster/config-defaults.yaml

```diff
@@ -82,9 +82,8 @@ skipper_suppress_route_update_logs: "true"
 skipper_validate_query: "true"
 skipper_validate_query_log: "false"
 
-# skipper default filters
 skipper_default_filters: 'enableAccessLog(4,5) -> fifo(2000,20000,"3s")'
-# inline routes
+skipper_disabled_filters: ""
```

Comment:
Named similar to `skipper_default_filters`, i.e. did not use `skipper_ingress_` prefix